### PR TITLE
Document package diff command with OpenAPI

### PR DESCRIPTION
**Pull request by eduardoj**
For reviewers, access directly to the documented endpoint in the review app through this [link](https://obs-reviewlab.opensuse.org/eduardoj-apidocs_newpackage_cmd_diff/apidocs-new/index#/Sources%20-%20Packages/post_source__project_name___package_name__cmd_diff).

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      description: Limit diff to the given file name.
+      example: hello_world.spec
+    - in: query
+      name: filelimit
+      schema:
+        type: string
+      description: |
+        Limit the diff size by line numbers.
+
+        By default it takes a built-in default value.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml:38]: Would be nice to know what that built-in default value is...

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    - in: query
+      name: rev
+      schema:
+        type: string
+      description: Revision of the package.
+    - in: query
+      name: tarlimit
+      schema:
+        type: string
+      description: Limit the diff size by line numbers.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml:101]:
```suggestion
      description: Limit the size of the diff of tar files to this amount of lines.
```

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+      schema:
+        type: string
+      description: Limit diff to the given file name.
+      example: hello_world.spec
+    - in: query
+      name: filelimit
+      schema:
+        type: string
+      description: |
+        Limit the diff size by line numbers.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml:36]:
```suggestion
        Limit the size of the diff to this amount of lines.
```

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+      name: tarlimit
+      schema:
+        type: string
+      description: Limit the diff size by line numbers.
+      example: 20
+    - in: query
+      name: unified
+      schema:
+        type: string
+      description: Set to `1` to use unified format without looking into archives.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml:107]: What are archives?

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml (hunk 3 above)
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml]: Applied.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml (hunk 1 above)
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml]: ...200. Changed.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml (hunk 2 above)
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml]: Applied.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml (hunk 4 above)
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml]: I rephrased the description. Please have a look.

**Review by danidoni — APPROVED** [review]

**Review by hennevogel — APPROVED** [review]

**Comment by hennevogel** [comment]
codecov hangs. coverage is fine. merging!

**Pull request merged**